src/main/java/com/dreammaster/TwilightForest/TF_Loot_Chests.java

```diff
@@ -15,96 +15,142 @@
-        TFTreasure.hill3.rare.add(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 2L), 7);
-        TFTreasure.hill3.rare.add(GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Aluminium, 1L), 5);
-        TFTreasure.hill3.rare.add(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Mica, 16L), 7);
-        TFTreasure.hill3.ultrarare.add(GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Gallium, 1L), 5);
-
-        //Litch Tower Basment
+        //UrGhast Tower Basment
         TFTreasure.basement.common.add(new ItemStack(Blocks.torch, 16, 0), 20);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 0), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 1), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 2), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 3), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 4), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 5), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 6), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 7), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 8), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 9), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 10), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 11), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 12), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 13), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 14), 5);
-        TFTreasure.basement.common.add(GT_ModHandler.getModItem("Thaumcraft", "blockCandle", 4, 15), 5);
-        TFTreasure.basement.uncommon.add(new ItemStack(Items.potionitem, 1, 8198), 7);//potion night vision 3:00
-        TFTreasure.basement.uncommon.add(GT_ModHandler.getModItem("Thaumcraft", "ItemResource", 2, 1), 5);
-        TFTreasure.basement.rare.add(new ItemStack(Items.potionitem, 1, 8262), 7);//night vision 8:00
-        TFTreasure.basement.ultrarare.add(GT_ModHandler.getModItem("Thaumcraft", "ItemResource", 1, 0), 5);
-        TFTreasure.basement.ultrarare.add(GT_ModHandler.getModItem("Thaumcraft", "ItemResource", 4, 14), 3);
+        TFTreasure.basement.common.add(GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.StainlessSteel, 16L), 50);
```

Comment:
Not sure when you kill the urGhast but i gues you mid end mv so you have setup your EBF allready for stainless steel. Sure i can lower the amount

Reply:
Why not right away? It's not like the bosses are hard, you can do it right after you make an assembler, can't you?

Reply:
A good rapier and a good crossbow is all you need up to this point.

Reply:
idk. If they so easy to kill i set them on always infernal list. I am mid mv and i got killed a lot of times in Litch tower. I not even try to kill hydra yet.

Reply:
That's why you climb the outside of the tower. There's waaay too many inside it, especially with the infernals and special mobs.

Reply:
maybe we can add some code that you cant climb up outside or you take damage maybe. Like you cant stand on blocks when play the mini game in the dungeon

Reply:
Then people just wouldn't do it, because infernals hyper-nerfed the lich tower. They'd just cheat instead. Also, dungeons don't have minigames in them. If you mean the loot games rooms, I still don't know what you mean, how would you do the minigame if you can't enter the room?